drivers/watchdog/Kconfig

```diff
@@ -30,4 +57,6 @@ source "drivers/watchdog/Kconfig.esp32"
 
 source "drivers/watchdog/Kconfig.sam0"
 
+source "drivers/watchdog/Kconfig.nrf"
```

Comment:
"Kconfig.nrfx" - that's the convention we decided to use for shims using nrfx drivers.